repeater/application.py

```diff
@@ -179,3 +187,30 @@ def _handle(self, request):
         response.content_type = 'text/plain'
         response.body = 'OK'
         return response
+
+    def check_remote_address(self, hosts, remote_address):
+        """
```

Comment:
I wouldn't say this method should be part of a public interface of the class. The convention is, to prepend the private name with underscore (```_```). 

This could make testing a bit harder - unit tests (in my opinion)  should specify behaviour of an unit (here the ```Request``` class) and therefore we should test only public interface (the ```__call__``` method). Fortunately the ```check_remote_address``` method does not depends on internal state of an instance, so it might be implemented as an utility (just like ```sign_request`` function above) and so it can be easily tested.

The same for ```return_address``` method below.